docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -205,6 +223,64 @@ export default class MethodPage extends React.PureComponent<Props, State> {
                     className: 'code',
                   }}
                 />
+                {TRANSPORTS.getDebugTransport(method) instanceof
```

Comment:
`instanceof` can work or not work based on webpack behavior, it's better to use dynamic dispatch of a method

Reply:
Didn't know about it. Thank you~!